public/app/features/correlations/Forms/ConfigureCorrelationSourceForm.tsx

```diff
@@ -84,27 +88,26 @@ export const ConfigureCorrelationSourceForm = () => {
             readOnly={readOnly}
           />
         </Field>
-
         {variables.length > 0 && (
-          <Card>
+          <Card className={styles.cardContainerBackground}>
             <Card.Heading>Variables used in the target query</Card.Heading>
             <Card.Description>
-              <div>
-                You have used following variables in the target query:{' '}
-                {variables.map((name, i) => (
-                  <span className={styles.variable} key={i}>
-                    {name}
-                    {i < variables.length - 1 ? ', ' : ''}
-                  </span>
-                ))}
-              </div>
-              <div>
-                A data point needs to provide values to all variables as fields or as transformations output to make the
-                correlation button appear in the visualization.
-              </div>
+              You have used following variables in the target query:{' '}
+              {variables.map((name, i) => (
+                <span className={styles.variable} key={i}>
+                  {name}
+                  {i < variables.length - 1 ? ', ' : ''}
+                </span>
+              ))}
+              <br />A data point needs to provide values to all variables as fields or as transformations output to make
+              the correlation button appear in the visualization.
+              <br />
+              Note: Not every variable needs to be explicitly defined below. A transformation such as{' '}
```

Comment:
I added this note because I could see someone wanting to do a 1-1 match with the listed variables here and the variable list they construct below, even if it doesn't make sense to do so.